### DISCUSSION TITLE
Annual pricing of Hubs-Cloud

### DISCUSSION THREAD

**Discussion by elmau** [mozilla/hubs#3292] [Hubs Cloud Questions] [answered]
Is it possible to provision an annual price for Hubs-Cloud? In AWS the EC2 have different savings plans, but I don't know how these can be applied to Hubs-Cloud. Or is it always on hourly costs?

**Comment by robinkwilson** [comment, marked as answer]
To be honest, this is more a question for AWS marketplace and using their resources than us as the software provider for those machines.

People have applied credits from AWS to the HC AWS resources.

I'd love to know what you find out. Hope this helps.
